src/core/component.js

```diff
@@ -109,12 +111,13 @@ Component.prototype = {
    * If component is single-property, then parses the single property value.
    *
    * @param {string} value - HTML attribute value.
+   * @param {boolean} silent - silence warnings
    * @returns {object} Component data.
    */
-  parse: function (value) {
+  parse: function (value, silent) {
```

Comment:
The `silent` arg seems a bit awkward/misplaced to me. I'm going to poke around the code to see why we need it